src/main/java/org/broadinstitute/hellbender/tools/sv/Pedigree.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import java.io.*;
+import java.util.*;
+
+public class Pedigree {
+
+    public enum Sex {
+        MALE("1"), FEMALE("2"), UNKNOWN("other");
+
+        /// stores PED format compatible format for to/from string conversions
+        private final String pedEncode;
+        Sex(String code) {
+            this.pedEncode = code;
+        }
+
+        public static Sex fromString(String sex) {
+            return switch (sex) {
+                case "1" -> MALE;
+                case "2" -> FEMALE;
+                default -> UNKNOWN;
+            };
+        }
+
+        @Override
+        public String toString() {
+            return pedEncode;
+        }
+    }
+
+    public enum Phenotype {
+        MISSING("0"), UNAFFECTED("1"), AFFECTED("2");
+
+        /// stores PED format compatible format for to/from string conversions
+        private final String pedEncode;
+        Phenotype(String code) {
+            this.pedEncode = code;
+        }
+
+        public static Phenotype fromString(String affected) {
+            return switch (affected) {
+                case "-9", "0" -> MISSING;
+                case "1" -> UNAFFECTED;
+                case "2" -> AFFECTED;
+                default -> throw new IllegalArgumentException("Invalid affected status: " + affected);
```

Comment:
This is a bit of an aside, typically in GATK when we have an error based on an invalid input file, we want to propagate that as a UserException of some sort ( UserException.MalformedFile is a classic). That way the user gets a clear message that their data is the problem and not a bug in the code.